whitelist-windows-defender.bat

```diff
@@ -0,0 +1,22 @@
+@echo off
+echo Windows Defender hates SharpMonoInjector, attempting to whitelist...
+
+net session >nul 2>&1
+if %errorlevel% neq 0 (
+    echo [91mERROR: This script must run as administrator.[0m
+    pause
+    exit /b -1
+)
+
+for %%p in ("%~dp0" "%TEMP%\.net\SharpMonoInjector") do (
+    powershell -ExecutionPolicy Bypass -Command "Add-MpPreference -ExclusionPath "%%p"" || goto :failed
+)
+
+echo [92mSuccess![0m
+pause
+exit /b 0
+
+:failed
+echo [91mERROR: Failed to whitelist.[0m
+pause
+exit /b 1
```

Comment:
I am terribly worried about adding this.. I think we should make this an opt-in conscious decision someone has to make and add this to docs instead.

Reply:
Well, it is technically opt-in, I didn't add it to the other bat files or anything. It's just a standalone script for whoever needs it.

Reply:
Yeah, but the average user is going to press on that blindly. I'll shift it over to the docs.

Reply:
I think having the bat directly in there is very convenient, otherwise Defender starts spewing plenty of blatantly false positives about SharpMonoInjector being a dozen different viruses

Reply:
In addition, if you click "Allow threat" in Windows Defender, it will actually whitelist the virus signature for all files, instead of just ignoring that file... I'm having trouble believing Microsoft devs are that much brain-dead.

Reply:
moved the script to submodules folder and mentioned in readme, should do the trick